ad7768-1fmcz/ad77681.h

```diff
@@ -522,28 +525,52 @@ struct ad77681_dev {
 	/* SPI */
 	spi_desc			*spi_desc;
 	/* Configuration */
-	enum ad77681_power_mode		power_mode;
-	enum ad77681_mclk_div		mclk_div;
-	enum ad77681_conv_mode 		conv_mode;
-	enum ad77681_conv_diag_mux 	diag_mux_sel;
-	bool 					conv_diag_sel;
-	enum ad77681_conv_len		conv_len;
-	enum ad77681_crc_sel 		crc_sel;
-	uint8_t					status_bit;
+	enum ad77681_power_mode			power_mode;
+	enum ad77681_mclk_div			mclk_div;
+	enum ad77681_conv_mode 			conv_mode;
+	enum ad77681_conv_diag_mux 		diag_mux_sel;
+	bool 							conv_diag_sel;
+	enum ad77681_conv_len			conv_len;
+	enum ad77681_crc_sel 			crc_sel;
+	uint8_t							status_bit;
```

Comment:
Extra tab has been added between type and variable on lines: 525 - 535.
They should probably remain untouched.